chapters/annotations.tex

```diff
@@ -1747,13 +1747,15 @@ \subsubsection{Conversion Rules}\label{conversion-rules}
 The conversion should generate correct Modelica models using the new version of the library
 corresponding to the old version.
 
+The string arguments for these functions should not be the empty string (except for the string in an empty array of strings).
+
 \begin{nonnormative}
 Whenever possible tools should preserve the original style of the model, e.g., use of imports.
 Conversions should be applied in all places where named element are used in code, including Modelica URIs (for example, in \lstinline!Documentation! annotations).
 \end{nonnormative}
 
-These functions can be called with literal strings or array of strings
-and vectorize according to \cref{scalar-functions-applied-to-array-arguments}.
+These functions can be called with literal strings or array of strings and vectorize according to \cref{scalar-functions-applied-to-array-arguments}.
```

Comment:
Maybe fix grammar as well?
```suggestion
These functions can be called with literal strings or arrays of strings and vectorize according to \cref{scalar-functions-applied-to-array-arguments}.
```

Reply:
The current formulation leaves the status of something like `{"a" + "b"}` unclear.  Is the real intention to support:
- Literal strings.
- Arrays of literal strings (for example, `{"a", "b"}`).
- Calls to `fill` with a literal string as element value (for example, `fill("", 0)`).

Given this, the meaning of the following would be clear:
- The empty literal string is only allowed when constructing an empty array using `fill`.